modules/ossm-threescale-manifests.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/threescale-adapter.adoc
+
+[id="ossm-threescale-manifests_{context}"]
+= Generating manifests from a deployed adapter
+To generate manifests from a deployed adapter, run the following command replacing the following arguments with the 3scale Admin Portal URL, Service ID, and access token:
```

Comment:
Please reformat this procedure to follow the template.

Reply:
Fixed.